packages/components/src/utils/frameProps.tsx

```
@@ -83,21 +83,21 @@ type TransientFrameProps = TransientProps<FrameProps>;
+export const pickAndPrepareFrameProps = <
+    TProps extends { [K in FramePropsKeys]?: FrameProps[K] },
+    KFP extends FramePropsKeys[],
+>(
+    props: TProps,
+    allowedFrameProps: KFP,
 ) => {
-    const selectedProps = allowedFrameProps.reduce(
+    const selectedProps = allowedFrameProps.reduce<{
+        [value in KFP[number]]: TProps[value];
+    }>(
         (acc, item) => ({ ...acc, [item]: props[item] }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)